inc/builder/type/header/woo-cart/dynamic-css/dynamic.css.php

```diff
@@ -498,6 +499,53 @@ function astra_hb_woo_cart_dynamic_css( $dynamic_css, $dynamic_css_filtered = ''
 		$css_output .= astra_parse_css( $header_cart_icon );
 	}
 
+	/**
+	 * Added for the Cart total label badge position
+	 */
+	
+	$css_total_position_common_selector = array(
+		'.cart-container, .ast-addon-cart-wrap' => array(
+			'display'     => 'flex',
+			'align-items' => 'center',
+		),
+		'.astra-icon'                           => array(
+			'line-height' => 0.1,
+		),
+	);
+	$css_total_position_output_bottom   = array(
+		'.cart-container.ast-cart-position-bottom, .ast-addon-cart-wrap.ast-cart-position-bottom' => array(
```

Comment:
```suggestion
		'.ast-cart-position-bottom, .ast-cart-position-bottom' => array(
```
@GauravKhupse @sagarb90 Can we reduce the CSS selectors weightage from dynamic CSS and preview JS as well?

Reply:
Right but then this will the single class here `.ast-cart-position-bottom,` @GauravKhupse any reason why we use two different parent class for same `ast-cart-position-bottom` class

Reply:
Changes done!